src/WebJobs.Extensions.DurableTask/LocalGrpcListener.cs

```diff
@@ -153,13 +153,33 @@ public override Task<P.DeleteTaskHubResponse> DeleteTaskHub(P.DeleteTaskHubReque
 
             public async override Task<P.CreateInstanceResponse> StartInstance(P.CreateInstanceRequest request, ServerCallContext context)
             {
+                var instance = new OrchestrationInstance
+                {
+                    InstanceId = request.InstanceId ?? Guid.NewGuid().ToString("N"),
+                    ExecutionId = Guid.NewGuid().ToString(),
+                };
+
+
                 try
                 {
-                    string instanceId = await this.GetClient(context).StartNewAsync(
-                        request.Name, request.InstanceId, Raw(request.Input));
+                        // we don't use a DurableOrchestrationClient here because it doesn't expose a "ScheduledStartTime" property
+                        await this.GetDurabilityProvider(context).CreateTaskOrchestrationAsync(
+                        new TaskMessage
+                        {
+                            Event = new ExecutionStartedEvent(-1, Raw(request.Input))
```

Comment:
I think you can use `request.Input` directly here. No need to wrap it.
```suggestion
                            Event = new ExecutionStartedEvent(-1, request.Input)
```

Reply:
As per Jacob's suggestion, I'll use the TaskHubClient instead to avoid regressing on Distributed Tracing